mta/fixtures/job_function.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+
+from conftest import generate_string
+from swagger_client.models.api_job_function import ApiJobFunction
+
+
+@pytest.fixture(scope="session")
+def jobFunction(create_api, get_api):
```

Comment:
Please follow the Python convention: lowercase letters for functions and vars, words are separated by underscore. The fixture's name should be: job_function

Reply:
Done. Thanks!